blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -60,7 +12,7 @@ const SmallManualPromo = ({ customFields = { showImage: true, showHeadline: true
   }
   return (
     <LazyLoad enabled={customFields.lazyLoad && !isAdmin}>
-      <SmallManualPromoItem customFields={{ ...customFields }} />
+      <SmallPromoPresentation customFields={customFields} {...customFields} />
```

Comment:
Why pass the custom fields and pass the custom fields as params?

Reply:
Just changed this so there is more clarity about what is being passed. `{...customFields}` has to be passed since that has props that help customize `small-manual-promo`, but i renamed the overall prop `customContent` since that is shared between the small promo blocks.